thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detector/email/filter/PrecisionRecallEvaluator.java

```diff
@@ -46,6 +50,7 @@ public PrecisionRecallEvaluator(AlertFilter alertFilter, List<MergedAnomalyResul
   private int notifiedNotLabeled;  // Anomaly is notified, but not labeled
   private int userReportTrueAnomaly; // Anomaly is user reported: true anomaly that was not sent out
   private int userReportTrueAnomalyNewTrend; // Anomaly is user reported: true anomaly new trend that was not sent out
+  private boolean isProjected = false;
```

Comment:
Can you add some comments describing the meaning and usage of isProjected.

Reply:
Added. thanks

Reply:
Usually the comment is before the variable, except the inline comment. Please put the comment ahead of the variable definition.